front/src/common/redux/reducers/studentRegistrationReducer.js

```diff
@@ -0,0 +1,30 @@
+import { REGISTRATION_STUDENT, REGISTRATION_STUDENT_ERROR, HIDE_STUDENT_REGISTERED_MODAL } from '../actions/types';
+
+const initialState = {
+  firstNameEngValid: false,
+  isFormSended: false,
+  isModalVisible: false,
+  errorMessage: '',
+};
+
+export function studentRegister(state = initialState, action) {
+  const { response } = action;
```

Comment:
Also, please, have in action objects only two fields:
- type
- payload (actual data should be stored here)